mmhuman3d/data/data_converters/surreal.py

```diff
@@ -218,8 +232,8 @@ def convert_by_mode(self, dataset_path: str, out_path: str,
                     keypoints2d_.append(keypoints2d)
                     keypoints3d_.append(keypoints3d)
                     bbox_xywh_.append(bbox_xywh)
-                    smpl['body_pose'].append(pose[3:, idx].reshape((23, 3)))
-                    smpl['global_orient'].append(pose[:3, idx])
+                    smpl['body_pose'].append(bpose[3:, ].reshape((23, 3)))
+                    smpl['global_orient'].append(bpose[:3, ])
                     smpl['betas'].append(beta[:, idx])
                     cam_param_.append(parameter_dict)
 
```

Comment:
I think for surreal, it is worth mentioning that the official annotation is buggy. And highlight in the docs our implementation if we have fixed the bugs.

Reply:
added in [602b963](https://github.com/open-mmlab/mmhuman3d/pull/143/commits/602b963cba19d6a022672225063a6941d225ac07)